candid_diff/design/candiff-CLI-tool.md

```diff
@@ -0,0 +1,89 @@
+# Tool design (`candiff`)
+
+The `candid_diff` package exposes the `candiff` tool for scripting Candid value comparisons that have human-readable error messages.
+
+The tool compares input and output canister messages to those recorded earlier, from earlier run(s).
+
+## Use cases
```

Comment:
Another use case is https://github.com/dfinity-lab/motoko/issues/1748, so that users can integrate a common test set into their own testing workflow.

Reply:
Good point.  Will update to reflect this.